app/views/layouts/govuk_admin_template.html.erb

```diff
@@ -121,6 +121,11 @@
 
           ga('create', 'UA-26179049-6', '<%= ENV.fetch('GOVUK_APP_DOMAIN') %>');
           ga('set', 'anonymizeIp', true);
+
+          <% if content_for?(:custom_dimension_js) %>
```

Comment:
Thinking about this a little more, `custom_dimension_js` might be better named as `before_pageview_js`